korebuild.json

```diff
@@ -1,10 +1,24 @@
 {
   "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/release/2.1/tools/korebuild.schema.json",
   "channel": "release/2.1",
+  "msbuildType": "full",
   "toolsets": {
     "nodejs": {
       "minVersion": "8.0",
       "required": true
+    },
+    "visualstudio": {
+      "required": [
+        "Windows"
+      ],
+      "includePrerelease": true,
+      "versionRange": "[15.8, 16.0)",
+      "requiredWorkloads": [
+        "Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Win81",
+        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.ATL",
+        "Microsoft.VisualStudio.Component.Windows10SDK.15063.Desktop"
+      ]
```

Comment:
@jkotalik @natemcmaster How necessary is this requirement on Windows? I don't have any of these workloads installed, and I don't want to install it.

Now I can't even run `.\build.cmd /t:restore` since KoreBuild complains I don't have a compatible version of MSBuild, so I have to revert this change to korebuild.json just to restore packages in this repo.

Can't we just skip building IISIntegration and any other projects that need these special VS workloads on Windows machines that don't have these workloads installed? We have to do this for non-Windows platforms anyway, right?

Reply:
> Can't we just skip building IISIntegration and any other projects that need these special VS workloads on Windows machines that don't have these workloads installed?

We discussed this in build triage, and consensus was that we'd rather have build.cmd fail fast than fail later with mysterious errors. We'll work on improving error messages to make it more obvious how to install VS with the required workloads. Tracking issue: https://github.com/aspnet/AspNetCore-Internal/issues/1505